national-resources/sweden.md

```diff
@@ -0,0 +1,121 @@
+- [COVID-19 test statistics from the National Pandemic Centre](https://www.covid19dataportal.se/dashboards/npc-statistics/): shows the number of tests conducted each week, and the relative proportions of positive, negative, and inconclusive results over time (no longer updated as of December 2020).
+- [The Swedish Public Health Agency’s COVID-19 statistics ](https://experience.arcgis.com/experience/09f821667ce64bf7be6f9f87457ed9aa): shows data on the number of cases, admissions to intensive care, and mortality related to COVID-19 over time (only available in Swedish).  
+- [The Swedish Public Health Agency’s information on COVID-19 PCR tests ](https://www.folkhalsomyndigheten.se/PCR-covid-19/): shows the number of PCR tests performed over time in Sweden (only available in Swedish).  
+- [The National Board for Health and  Welfare’s dashboard on COVID-19 statistics](https://www.socialstyrelsen.se/statistik-och-data/statistik/statistik-om-covid-19/): shows information on the number of COVID-19 cases, and survival rates, as well as information on post COVID-19 condition/longcovid both for different regions and over time (only available in Swedish).
+- [The National Board for Health and  Welfare’s dashboard on data indirectly related to COVID-19](https://www.socialstyrelsen.se/statistik-och-data/statistik/statistik-om-covid-19/statistik-relaterad-till-covid-19/): shows information related to the number of influenza cases over time, as well as general mortality rates in Sweden (only available in Swedish and no longer updated as of January 2022).
+
+## National data sources
+<!--- A section to list and provide context to national data sources. Data source as defined in BY-COVID - those are online resources in which data can be obtained. --->
+- [Database of Swedish COVID-19 publications that make data available](https://www.covid19dataportal.se/datasets/all/). This resource lists all publications on COVID-19 that involve at least one researcher affiliated with a Swedish research institution and make their data openly available. The data has been categorised according to data type (e.g. social and humanities data, genomics data) to aid users (data is openly available).
+- [COVID-19 data sources from The Public Health Agency of Sweden](https://www.folkhalsomyndigheten.se/smittskydd-beredskap/utbrott/aktuella-utbrott/covid-19/statistik-och-analyser/) (data is openly available).
+- [COVID-19 data sources from The National Board for Health and  Welfare](https://www.socialstyrelsen.se/statistik-och-data/statistik/statistik-om-covid-19/) (data is openly available).
+- [Infectious diseases data sources published on the SciLifeLab Data Repository](https://figshare.scilifelab.se/categories/Biomedical_and_clinical_sciences/24457?categories=24508) (some data is openly available, otherwise access to data is restricted).
```

Comment:
```suggestion
- [Infectious diseases data sources published on the SciLifeLab Data Repository](https://figshare.scilifelab.se/categories/Biomedical_and_clinical_sciences/24457?categories=24508) (some data is openly available, otherwise access to data is restricted). The SciLifeLab Data Repository includes multiple different types of data (e.g. image data), the information found at the link shows only the data related to infectious disease, regardless of data type. 
```